package.json

```diff
@@ -38,8 +38,9 @@
 		"gettext-parser": "^4.0.3",
 		"husky": "^3.0.2",
 		"identity-obj-proxy": "^3.0.0",
-		"jest": "^24.9.0",
+		"jest": "26.6.3",
 		"jest-junit": "^6.3.0",
+		"jest-snapshot": "^27.4.5",
```

Comment:
> I see that this dependency was added to `e2e-tests` package ([reference](https://github.com/WordPress/gutenberg/pull/37396/files#diff-73662dce866d908da409d58c0dd1b3fcef7016ad2125dbb7b01a4172249f5280R35)) that AFAIK we're not using it in GB-mobile, do we need it even though?

@fluiddot I agree that we do not leverage `jest-snapshot` directly and it is odd that we would need to add it as a direct dependency. However, without adding `"jest-snapshot": "^27.4.5"`, the CI tasks currently fail. 

The failure without the addition seems to be related to dependency resolution. For example, without this addition, the test scripts resolve `node_modules/jest-jasmine2@26.6.3 > gutenberg/node_modules/jest-snapshot@27.4.5`. This results in the error captured in the PR description, as `jest-snapshot@27` includes a breaking change of a newly returned `Promise`: https://github.com/facebook/jest/pull/8829

Adding `"jest-snapshot": "^27.4.5"` resolved the error, but it may be an inappropriate solution. It would appear that the resolution path is now `node_modules/jest-jasmine2@26.6.3 > node_modules/jest-snapshot@27.4.5/node_modules/jest-snapshot@26.6.3`, meaning that `node_modules/jest-snapshot@27.4.5` is unused. 

It is all quite strange. I am researching this further.

Reply:
> @fluiddot I agree that we do not leverage jest-snapshot directly and it is odd that we would need to add it as a direct dependency. However, without adding "jest-snapshot": "^27.4.5", the CI tasks currently fail.

Oh I see, in that case, let's stick with this approach until we found a better alternative 👍 .